packages/core/content-releases/server/src/routes/release.ts

```diff
@@ -65,5 +65,21 @@ export default {
         ],
       },
     },
+    {
+      method: 'POST',
+      path: '/:id/publish',
```

Comment:
I wasn't aware of the convention in the CM and I'm not sure I completely understand it. 

At first I thought actions was referring to controller actions. But not all POST requests have actions (ie /collection-types/:model, /collection-types/:model/clone/:sourceId), it looks like there is some correlation with bulk actions, but it is also used for publish and unpublish. 

As Fernando mentioned content releases has a concept of actions (releases have many actions). So I think it could get confusing. Some of our routes, like create an action for a release use the word actions (POST /:releaseId/actions). 

Is this a convention we are trying to make consistent throughout the entire app? If not and it remains specific to the content manager I think I would prefer to leave the routes as they are for content releases.

Reply:
I'd probably have to defer to @Marc-Roig who's away today. I don't think it matters a whole lot & if you have `actions` already then i'd probably leave it 🤓